.charts/overwhelm/values.yaml

```diff
@@ -3,10 +3,10 @@ overwhelm:
   deployment:
     image:
       manager:
-        name: ghcr.io/expediagroup/overwhelm
+        repository: ghcr.io/expediagroup/overwhelm
         tag: latest
       kubeRbacProxy:
-        name: gcr.io/kubebuilder/kube-rbac-proxy
+        repository: gcr.io/kubebuilder/kube-rbac-proxy
```

Comment:
You also need to update https://github.com/ExpediaGroup/overwhelm/blob/af6f3ff71a5c9f533c56e15738df49ae98db0679/.charts/overwhelm/templates/overwhelm-controller-manager-deployment.yaml#L50

Reply:
addressed ( I believe! )